CHANGELOG.md

```diff
@@ -2,6 +2,9 @@ Tantivy 0.23 - Unreleased
 ================================
 Tantivy 0.23 will be backwards compatible with indices created with v0.22 and v0.21. The new minimum rust version will be 1.75.
 
+> [!WARNING]
```

Comment:
that's already outdated with the last release

https://github.com/quickwit-oss/tantivy/pull/2621/files

Reply:
I'm not sure I understand: I didn't update the lines linked in that pr so even if we do 0.24 I'm not sure I see the problem. 

I could rebase but I don't think that's what you mean because that pr isn't merged.

Reply:
0.24 is already released, but the CHANGELOG pr is not yet merged